test/extended/cli/explain.go

```diff
@@ -344,6 +381,53 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 
 	oc := exutil.NewCLI("oc-explain")
 
+	g.It("list uncovered GroupVersionResources", func() {
+		resourceMap := make(map[schema.GroupVersionResource]bool)
+		kubeClient := kclientset.NewForConfigOrDie(oc.AdminConfig())
+		_, resourceList, err := kubeClient.Discovery().ServerGroupsAndResources()
+		if err != nil {
+			e2e.Failf("Failed reading groups and resources %v", err)
+		}
+		for _, rl := range resourceList {
+			for _, r := range rl.APIResources {
+				gv, err := schema.ParseGroupVersion(rl.GroupVersion)
+				if err != nil {
+					e2e.Failf("Couldn't parse GroupVersion for %s: %v", gv, err)
+				}
+				resourceMap[gv.WithResource(r.Name)] = true
+			}
+		}
+
+		for _, bt := range builtinTypes {
+			delete(resourceMap, bt)
+		}
+		for _, ct := range crdTypes {
+			delete(resourceMap, ct)
+		}
+		for _, st := range specialTypes {
+			resource := strings.Split(st.field, ".")
+			delete(resourceMap, st.gv.WithResource(resource[0]))
+		}
+		for _, snt := range specialNetworkingTypes {
+			resource := strings.Split(snt.field, ".")
+			delete(resourceMap, snt.gv.WithResource(resource[0]))
+		}
+
+		e2e.Logf("These GroupVersionResources are missing proper explain test:")
+		for k := range resourceMap {
+			// ignore all k8s built-ins and sub-resources
+			if k.Group == "" || strings.Contains(k.Group, "k8s.io") ||
+				k.Group == "apps" || k.Group == "autoscaling" ||
+				k.Group == "batch" || k.Group == "extensions" ||
+				k.Group == "policy" ||
+				strings.Contains(k.Group, "cncf.io") ||
+				strings.Contains(k.Resource, "/") {
+				continue
+			}
+			e2e.Logf(" - %s", k)
```

Comment:
This is currently not failing tests, but a helper to figure out what we miss. With this PR it's 0, so I'm considering turning that into a failing test, but that's an open question.